xml/admin_docupdates.xml

```diff
@@ -155,7 +155,7 @@
      <itemizedlist>
       <listitem>
        <para>Fixed information about encrypted communication and added
-        <xref linkend="vnc.encrypted"/> on how to set it up
+        <xref linkend="sec.vnc.encrypted"/> on how to set it up
```

Comment:
not completely sure here... "sec.vnc.encrypt" would use the verb in infinitive. however, "encrypted" can either be a verb or an adjective...